templates/web/smidsy/maps/osm-toner-lite.html

```diff
@@ -0,0 +1,15 @@
+[% map_js = BLOCK %]
+<script type="text/javascript" src="[% version('/js/OpenLayers.fixmystreet.js') %]"></script>
+<script type="text/javascript" src="https://stamen-maps.a.ssl.fastly.net/js/tile.stamen.js?v1.3.0"></script>
+<script type="text/javascript" src="[% version('/js/map-OpenLayers.js') %]"></script>
+<script type="text/javascript" src="[% version('/js/map-toner-lite.js') %]"></script>
+<script type="text/javascript" src="[% version('/js/jquery.ba-hashchange.min.js') %]"></script>
+<!--[if lte IE 6]>
+  <link rel="stylesheet" href="/js/OpenLayers-2.13.1/theme/default/ie6-style.css" type="text/css">
+<![endif]-->
+[% END %]
+
+[% map_html = BLOCK %]
+[% INCLUDE maps/openlayers.html %]
+<script type="text/javascript">fixmystreet.show_stats19 = '[% show_stats19 %]';</script>
```

Comment:
Seems a shame to have to override this whole template to get the flag into the JS. Could it be done in a smaller template?

There's a [XSS hole](http://smidsy.fixmystreet.staging.mysociety.org/around?pc=np203qn&show_stats19=1%27-alert(%27hello%27)-%27) here too, that value needs escaping or sanitising server-side.
Might be worth prefixing this line with `fixmystreet = fixmystreet || {}` to avoid a potential crash.